### ISSUE TITLE
Allow artist tags to implicate meta tags

### ISSUE THREAD

**Issue by Mokou** [danbooru/danbooru#5275]
When discussing the issue of Danbooru images being used for AI art training [I suggested](https://danbooru.donmai.us/forum_posts/223089) that a meta tag such as `do_not_train` be created for implication from artist tags for those that don’t want to be used for training AI models. However it seems that cross-category implications have been disabled in danbooru@4a4c198.

I think it could be useful to allow for this type of implication, specifically in this case, but potentially for other cases as well.

**Comment by luni3359** [comment]
I wonder why it was changed. I feel like it's a good feature to have in the grand scheme of things.

**Comment by nottalulah** [comment]
I believe the issue was people kept requesting dumb implications like some_pokemon -> gen_x_pokemon or character -> elf. So the ability to request them at all was removed.

**Comment by Mokou** [comment]
Yeah, that makes sense. I wouldn't think those sorts of requests would a problem if only artist -> meta implications were re-enabled.
